packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -2898,36 +2893,42 @@ describe('TransactionController Integration', () => {
           ],
         });
 
-        const firstNonceLock = await transactionController.getNonceLock(
+        const firstNonceLockPromise = transactionController.getNonceLock(
           ACCOUNT_MOCK,
           networkClientId,
         );
+        await advanceTime({clock, duration: 1})
+
+        const firstNonceLock = await firstNonceLockPromise
```

Comment:
Just curious what was this fixing?

Reply:
the fake timer must be advanced before the promise from getNonceLock will resolve